src/components/TrendChartDetails.js

```diff
@@ -38,9 +38,14 @@ const TrendChartDetails = ({ colors, crime, data, dispatch, keys }) => {
       <div className='flex-auto'>
         <h4 className='mt0 mb1 fs-18 sans-serif'>{year}</h4>
         <p className='mb1 lg-m0 lg-pr4 lg-mh-72p fs-14 sm-fs-16'>
-          {name}’s {term} rate was {comparison} that of the United States, and
-          in {highlight(year)} was {highlight(formatRate(rate))} incidents
-          per 100,000 people.
+          {keys.length === 1 && (<span>
```

Comment:
can you tweak this to be a variable (just for clarity sake), like `isOnlyNational`, which we can use on L44 too?